Dockerfile

```
@@ -49,12 +46,9 @@ RUN mkdir -p /opt/src/pypgstac
 
 WORKDIR /opt/src/pypgstac
 
-COPY pypgstac/poetry.lock pypgstac/pyproject.toml ./
-RUN poetry install
-
-
 COPY pypgstac /opt/src/pypgstac
-RUN poetry install
+
+RUN pip3 install -e /opt/src/pypgstac
```

```suggestion
RUN python3 -m pip install -e /opt/src/pypgstac
```

*might* be the cause of the error you're seeing in CI.